pkg/pipelines/argocd/argocd.go

```diff
@@ -38,9 +38,8 @@ func Build(argoNS, repoURL string, m *config.Manifest) (res.Resources, error) {
 	if repoURL == "" {
 		return res.Resources{}, nil
 	}
-	argoEnv, err := m.GetArgoCDEnvironment()
-	// If there's no ArgoCD environment, then we don't need to do anything.
-	if err != nil {
+	argoEnv, err := m.GetArgo()
```

Comment:
You're not using `err` here, but you can remove this, as it now returns `nil` if it doesn't exist, and there can't be any more than one now.